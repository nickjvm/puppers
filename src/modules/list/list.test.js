import { ListPuppers } from './index';
import React from 'react';
import renderer from 'react-test-renderer';

// beforeAll(() => {
//     const mockGeolocation = {
//         getCurrentPosition: jest.fn(),
//         watchPosition: jest.fn()
//     };

//     global.window.navigator.geolocation = mockGeolocation;
// })

describe('list', () => {
    it('renders a list of pets', () => {
        ListPuppers.fetchData = jest.fn();
        console.log(ListPuppers.fetchData)
        const component = renderer.create(
            <ListPuppers pets={[{ id: '1', name: 'nick', image: 'an iced tea' }]} />
        )
        // component.getInstance().fetchData = jest.fn();
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})