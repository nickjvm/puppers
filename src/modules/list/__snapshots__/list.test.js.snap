// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list renders a list of pets 1`] = `
<div>
  <br />
  <form
    onSubmit={[Function]}
  >
    <input
      onChange={undefined}
      placeholder="Enter your zip code here!"
      type="search"
      value={undefined}
    />
  </form>
  <ul>
    <li>
      <div>
        nick
      </div>
      <div>
        <img
          src="an iced tea"
        />
      </div>
    </li>
  </ul>
</div>
`;
